ckanext/canada/templates/public/package/read.html

```diff
@@ -312,6 +312,14 @@
     });
   }
 });
+
+</script>
+<script src="https://cdn.datatables.net/1.10.13/js/jquery.dataTables.min.js"></script>
```

Comment:
we're not supposed to be using the cdn for the plain version of datatables. instead enable the one already included with WET: https://wet-boew.github.io/v4.0-ci/demos/tables/tables-en.html